docs/interpreter/spark-interpreter-k8s.md

```diff
@@ -0,0 +1,164 @@
+---
+layout: page
```

Comment:
We could put this in `spark.md` or at least add reference link to this doc in `spark.md`

Reply:
Added a link to this page in spark.md.